server/src/io.rs

```diff
@@ -76,9 +91,17 @@ pub struct IoRead {
 pub struct WriterComm {
     tx: flume::Sender<bytes::Bytes>,
     enc: PacketEncoder,
+
+    /// Approximate speed that the other side can receive the data that this sends.
+    /// Measured in bytes/second.
+    speed: Arc<AtomicU32>,
 }
 
 impl WriterComm {
+    pub fn speed(&self) -> u32 {
```

Comment:
```suggestion
    pub fn speed_mib(&self) -> u32 {
```

or `mib_per_s` potentially? or could also make this function still be `speed` make this return a type 

struct Speed(u32) (or similar)

and have method mib_per_second()

Reply:
b7c458c